src/main/java/frc/robot/commands/AutoCenter.java

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import com.ctre.phoenix6.mechanisms.swerve.SwerveRequest;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.LimelightHelpers;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.SwerveSubsystem;
+
+public class AutoCenter extends Command {
+  
+  boolean end = false;
+  double kP = -0.05;
+  private final Timer timer = new Timer();
+  private final SwerveSubsystem driveSubsystem;
+  private final SwerveRequest.FieldCentric drive;
+
+  private final ShootCommand shootCommand = new ShootCommand(RobotContainer.shooterSubsystem, RobotContainer.intakeSubsystem);
+
+  private final PrepCommand prep = new PrepCommand(RobotContainer.shooterSubsystem, 45, 50, true);
+  
+  /** Creates a new AutoCenter. */
+  public AutoCenter(SwerveSubsystem driveSubsystem, SwerveRequest.FieldCentric drive) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    this.driveSubsystem = driveSubsystem;
+    this.drive = drive;
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    
+    timer.start();
```

Comment:
I may not fully understand it, but why do you start the timer here? It seems reasonable that a reset might be what you want, but I could be not totally following the logic flow.